share/spice/indego_bikeshare_locations/indego_bikeshare_locations.js

```diff
@@ -0,0 +1,61 @@
+(function (env) {
+    "use strict";
+
+    env.ddg_spice_indego_bikeshare_locations = function(api_result) {
+        if (!api_result || !api_result.features) {
+            return Spice.failed('indegobikesharelocations');
```

Comment:
Match this ID with the IA page ID as well https://duck.co/ia/view/indego_bikeshare_locations